msteams-platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab.md

```diff
@@ -77,7 +77,7 @@ The following table provides the response codes:
 
 ## Limitations
 
-* Share to Teams is supported for desktop and web clients but isn't supported for mobile client.
+* Share to Teams is supported for desktop client but isn't supported for web and mobile clients.
```

Comment:
The Share to Teams feature is available for the desktop client, but it's not supported on web and mobile clients.

Reply:
Updated.